cmd/deployment/show.go

```diff
@@ -32,7 +35,10 @@ const showExample = `
   ecctl deployment show <deployment-id> --kind kibana
 
 * Shows apm resource information from a given deployment with a specified ref-id.
-  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server`
+  ecctl deployment show <deployment-id> --kind apm --ref-id apm-server
+
+* Return the current deployment state as a valid update payload.
+  ecctl deployment show <deployment id> --generate-payload > update.json`
```

Comment:
Does this line need a leading "`"?

Reply:
ah no, the enclosing with backpacks in Golang means a string literal with line breaks allowed in there, basically it allows us to define multi-line literals in a var.